dockerRunner.go

```diff
@@ -898,7 +937,7 @@ func (dr *dockerRunnerImpl) generateEntrypointScript(shell string, commands []st
 		shell,
 		firstCommands,
 		lastCommand,
-		strings.Replace(lastCommand, "'", "\\'", -1),
```

Comment:
Turned out that you can't really properly escape single quotes, so back to escaping double quotes and echo them inside double quotes.